### PR TITLE
ADM-669: [frontend] feat: update notification

### PR DESCRIPTION
**Pull request by JiangRu1**
## Summary

...

## Before

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

**Review by lxuebing — APPROVED** [review]

File: frontend/src/components/Common/NotificationButton/index.tsx [hunk 1]
```diff
@@ -1,42 +1,66 @@
...
+        iconColor: '#D78D20',
+        backgroundColor: '#FFF4E3',
+      };
+    case 'info':
+    default:
+      return {
+        icon: InfoIcon,
+        iconColor: '#4050B5',
+        backgroundColor: '#E9ECFF',
+      };
```
Comment (guzhongren) [inline frontend/src/components/Common/NotificationButton/index.tsx:35]: 主题色放在theme里

File: frontend/src/components/Common/NotificationButton/style.tsx [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
   },
-  zIndex: Z_INDEX.TOOLTIP,
-};
+  '& .MuiAlert-icon': {
+    color: props.iconcolor,
+  },
+}));
+
+export const AlertTitleWrapper = styled(AlertTitle)({
+  color: '#000000D9',
```
Comment (guzhongren) [inline frontend/src/components/Common/NotificationButton/style.tsx:24]: #000000D9 从theme里取， 不正确的值

File: frontend/src/components/Metrics/ConfigStep/index.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ import { useNotificationLayoutEffectInterface } from '@src/hooks/useNotification
 
 const ConfigStep = ({ resetProps }: useNotificationLayoutEffectInterface) => {
   useLayoutEffect(() => {
-    resetProps?.();
+    resetProps();
```
Comment (guzhongren) [inline frontend/src/components/Metrics/ConfigStep/index.tsx:9]: React Hook useLayoutEffect has a missing dependency: 'resetProps'. Either include it or remove the dependency array. If 'resetProps' changes too often, find the parent component that defines it and wrap that definition in useCallback

 Check warning on line 10 in frontend/src/components/Metrics/ConfigStep/index.tsx

GitHub Actions
/ frontend-check

React Hook useLayoutEffect has a missing dependency: 'resetProps'. Either include it or remove the dependency array. If 'resetProps' changes too often, find the parent component that defines it and wrap that definition in useCallback





  return (

    <ConfigStepWrapper>

File: frontend/src/components/Metrics/ReportStep/index.tsx [hunk 4]
```diff
@@ -82,14 +86,19 @@ const ReportStep = ({ notification, handleSave }: ReportStepProps) => {
         clearInterval(timer);
       };
     }
-  }, [exportValidityTimeMin]);
+  }, [exportValidityTimeMin, isAllMetricsReady]);
+
+  useLayoutEffect(() => {
+    resetProps();
+  }, [pageType]);
```
Comment (guzhongren) [inline frontend/src/components/Metrics/ReportStep/index.tsx:93]: React Hook useLayoutEffect has a missing dependency: 'resetProps'. Either include it or remove the dependency array.

File: frontend/src/components/Common/NotificationButton/index.tsx (hunk 1 above)
Comment (JiangRu1) [reply frontend/src/components/Common/NotificationButton/index.tsx]: done

File: frontend/src/components/Common/NotificationButton/style.tsx (hunk 2 above)
Comment (JiangRu1) [reply frontend/src/components/Common/NotificationButton/style.tsx]: done

File: frontend/src/components/Metrics/ConfigStep/index.tsx (hunk 3 above)
Comment (JiangRu1) [reply frontend/src/components/Metrics/ConfigStep/index.tsx:9]: done

File: frontend/src/components/Metrics/ReportStep/index.tsx (hunk 4 above)
Comment (JiangRu1) [reply frontend/src/components/Metrics/ReportStep/index.tsx]: done

**Pull request merged**